src/libs/actions/Task.js

```diff
@@ -274,22 +274,6 @@ function completeTask(taskReport, taskTitle) {
         },
     ];
 
-    // Multiple report actions can link to the same child. Both share destination (task parent) and assignee report link to the same report action.
-    // We need to find and update the other parent report action (in assignee report). More info https://github.com/Expensify/App/issues/23920#issuecomment-1663092717
-    const assigneeReportAction = ReportUtils.getTaskParentReportActionIDInAssigneeReport(taskReport);
-    if (!_.isEmpty(assigneeReportAction)) {
-        const optimisticDataForClonedParentReportAction = ReportUtils.getOptimisticDataForParentReportAction(
-            taskReportID,
-            completedTaskReportAction.created,
-            CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
-            assigneeReportAction.reportID,
-            assigneeReportAction.reportActionID,
-        );
-        if (!_.isEmpty(optimisticDataForClonedParentReportAction)) {
-            optimisticData.push(optimisticDataForClonedParentReportAction);
-        }
-    }
-
```

Comment:
We still want this, we just don't want to update the childVisibleActionCount

Reply:
@thienlnam Currently when we close a task, two report actions are created.
1. This is the system message that gets added to the `taskReport`.
2. This is the `reportAction` that gets added to the `assigneeReport`.

I'm sorry for confusing it with `childVisibleActionCount` but actually we don't really need the 2nd `reportAction` because it seems it isn't getting added on the backend as well and moreover, if you see what is the purpose of adding a `reportAction` to the assignee report when we don't want to count it as a reply or do anything else with it?

Reply:
@thienlnam Moreover, I tried to find any discrepancy that can occur if we don't add the 2nd report action, but it seems to work like expected... If you come across an inconsistency that is happening if we don't add the 2nd report action, please let me know

Reply:
We're not creating a new reportAction that is added on the assigneeReport, we're just updating the linked reportAction in the assignee report.

If you take a look at updateOptimisticParentReportAction, you'll see it just updates some parameters of the parent reportAction (which is why it updated the comment count and such here). We don't want to update the childVisibleActionCount because that's causing the bug but we should keep the other things such as childCommenterCount as it's still another user adding the system message.

We can probably stop updating the childLastVisibleActionCreated and childVisibleActionCount, but let's keep the other two parameters

Reply:
@thienlnam 
Yes, I should have said the assignee `reportAction` is being updated instead of saying a new one is created. But still I don't think we should update it.
Here are the parameters that are currently being updated:
- `childVisibleActionCount`
- `childLastVisibleActionCreated`
- `childCommentorCount`
- `childOldestFourAccountIDs`

We have agreed that we don't need to update the first two. But if we look at the other 2 options, they are only updated on the front-end. So if we need to update the last two options, we also need to update them on the back-end.

**Front-end `reportAction`**

<img width="469" alt="Screenshot 2023-09-30 at 7 44 05 PM" src="https://github.com/Expensify/App/assets/77237602/0029cb10-55a7-4c13-8c43-0e817db31e1a">

<p></p>

**`reportAction` after signing out and back in**

<img width="469" alt="Screenshot 2023-09-30 at 7 47 08 PM" src="https://github.com/Expensify/App/assets/77237602/359340aa-c4ea-4fcb-ad2c-ddc3e29624de">

You can see from the screenshots that those properties just don't exist on the backend version of the `reportAction`. If you are making the argument that the system function should also be counted as a comment by the user, then it seems inconsistent that we add the user as a commentator in `childCommentorCount` but we don't increment the `childVisibleActionCount` while closing/re-opening.

But anyway, I have no problem updating the last two options as well, but then we would also have to update the code on the back-end for these changes as well.
Let me know if we are ready to make the required changes on the backend, and I'll make the required changes here.

cc @eVoloshchak (would like to hear your thoughts on this as well)

Reply:
Okay thanks for that explanation - in that case I agree with your analysis. If system messages aren't counted as updating the other parameters then we don't need to update those in the front end as well